sql/clients_last_seen_v1.init.sql

```diff
@@ -1,17 +1,20 @@
+WITH input AS (
+  SELECT
+    ROW_NUMBER() OVER (PARTITION BY client_id ORDER BY submission_date_s3 DESC) AS _row,
```

Comment:
I'm used to seeing `ROW_NUMBER() ... AS n`, although I do see that the leading underscore is a nice hint that we're planning to throw this column away.